public/app/constants/profile-metrics.json

```diff
@@ -0,0 +1,135 @@
+    "description": "Total count of blocking contentions",
+    "type": "contentions",
+    "group": "block",
+    "unit": "short"
+  },
+  "block:delay:nanoseconds:contentions:count": {
+    "id": "block:delay:nanoseconds:contentions:count",
+    "description": "Total nanoseconds spent in blocking delays",
+    "type": "delay",
+    "group": "block",
+    "unit": "ns"
+  },
+  "goroutine:goroutine:count:goroutine:count": {
+    "id": "goroutine:goroutine:count:goroutine:count",
+    "description": "Current number of goroutines",
+    "type": "goroutine",
+    "group": "goroutine",
+    "unit": "short"
+  },
+  "memory:alloc_in_new_tlab_bytes:bytes::": {
+    "id": "memory:alloc_in_new_tlab_bytes:bytes::",
+    "description": "Total bytes allocated inside Thread-Local Allocation Buffers (TLAB)",
+    "type": "alloc_in_new_tlab_bytes",
+    "group": "memory",
+    "unit": "bytes"
+  },
+  "memory:alloc_in_new_tlab_objects:count::": {
+    "id": "memory:alloc_in_new_tlab_objects:count::",
+    "description": "Number of objects allocated inside Thread-Local Allocation Buffers (TLAB)",
+    "type": "alloc_in_new_tlab_objects",
+    "group": "memory",
+    "unit": "short"
+  },
+  "memory:alloc_objects:count:space:bytes": {
+    "id": "memory:alloc_objects:count:space:bytes",
+    "description": "Total objects allocated, represented in bytes",
+    "type": "alloc_objects",
+    "group": "memory",
+    "unit": "count"
+  },
+  "memory:alloc_space:bytes:space:bytes": {
+    "id": "memory:alloc_space:bytes:space:bytes",
+    "description": "Total bytes allocated in the heap",
+    "type": "alloc_space",
+    "group": "memory",
+    "unit": "bytes"
+  },
+  "memory:inuse_objects:count:space:bytes": {
+    "id": "memory:inuse_objects:count:space:bytes",
+    "description": "Count of objects currently in use",
+    "type": "inuse_objects",
+    "group": "memory",
+    "unit": "short"
+  },
+  "memory:inuse_space:bytes:space:bytes": {
+    "id": "memory:inuse_space:bytes:space:bytes",
+    "description": "Total bytes of memory currently in use",
+    "type": "inuse_space",
+    "group": "memory",
+    "unit": "bytes"
+  },
+  "mutex:contentions:count:contentions:count": {
+    "id": "mutex:contentions:count:contentions:count",
+    "description": "Number of observed mutex contentions",
+    "type": "contentions",
+    "group": "mutex",
+    "unit": "short"
+  },
+  "mutex:delay:nanoseconds:contentions:count": {
+    "id": "mutex:delay:nanoseconds:contentions:count",
+    "description": "Total time spent waiting due to mutex contentions",
+    "type": "delay",
+    "group": "mutex",
+    "unit": "ns"
+  },
+  "process_cpu:alloc_samples:count:cpu:nanoseconds": {
+    "id": "process_cpu:alloc_samples:count:cpu:nanoseconds",
+    "description": "Count of memory allocation samples during CPU time",
+    "type": "alloc_samples",
+    "group": "memory",
+    "unit": "short"
+  },
+  "process_cpu:alloc_size:bytes:cpu:nanoseconds": {
```

Comment:
Didn't know this was a thing.

Reply:
yeah its unfortunate that these are so messy but haven't found a good solution yet for standardizing outside of this source-of-truth json file